service/__init__.py

```diff
@@ -4,17 +4,28 @@
 import dateutil
 import dateutil.parser
 import json
+from flask_login import LoginManager
+from flask_wtf.csrf import CsrfProtect
 
 from config import CONFIG_DICT
 
 
 app = Flask(__name__)
 app.config.update(CONFIG_DICT)
+app.secret_key = 'change_this_key-not_secure'
```

Comment:
can we add a TODO code comment here so we don't lose track that this needs to be fixed before going to prod